api/handlers.go

```diff
@@ -263,6 +264,17 @@ func (api *API) sweepStatusGET(w http.ResponseWriter, _ *http.Request, _ httprou
 	api.WriteJSON(w, api.staticSweeper.Status())
 }
 
+// scannerStatusGET responds with the status of the latest scan.
+func (api *API) scannerStatusGET(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	verbose, _ := strconv.ParseBool(r.FormValue("verbose"))
```

Comment:
I'm usually in favour of forcing the parsing to succeed if an optional flag is specified to help users immediately see potential mistakes they made on their end. e.g. a typo

Reply:
Yes, I generally agree with that as well. In this case it felt unnecessary but maybe I'm just being lazy. I'll add it with the next PR.